src/main/resources/static/styles.css

```
@@ -0,0 +1,92 @@
+.dropdown {
+    position: fixed;
+    top: 10px; /* Justera vertikalt avstånd från toppen */
+    right: 10px; /* Justera horisontellt avstånd från höger */
+}
+
+.dropdown-content {
+    display: none;
+    position: fixed; /* Ändra till relativ position */
+    top: 30px; /* Justera vertikalt avstånd från förälderelementet */
+    right: 0; /* Placera rutan helt till höger */
+    background-color: #f9f9f9;
+    min-width: 120px;
+    box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+    z-index: 1;
+}
+
+.dropdown-content a {
+    color: black;
+    padding: 12px 16px;
+    text-decoration: none;
+    display: block;
+}
+
+.dropdown-content a:hover {
+    background-color: #ddd;
+}
+
+.dropdown:hover .dropdown-content {
+    display: block;
+}
+
+/* Main table styles */
+.styled-table {
+    border-collapse: collapse;
+    margin: 5px auto;
+    font-size: 0.9em;
+    font-family: sans-serif;
+    min-width: 800px;
```

```suggestion
    width: 80vw; (or whatever value that looks best )
```
I think we should use vw as width because it is determined by the viewport. Making it look better on different platforms.